modules/admin_manual/pages/enterprise/user_management/saml_2.0_sso.adoc

```diff
@@ -53,17 +55,17 @@ php apps/user_shibboleth/tools/adfs2fed.php \
 Next, you need to configure `shibd`.
 To do this, in `/etc/shibboleth/shibboleth2.xml`:
 
-=== Use the URL of the ownCloud instance as the `entityID` in the `ApplicationDefaults`, e.g.,
+=== Use the URL of the ownCloud instance as the `entityID` in the `ApplicationDefaults`
```

Comment:
Can this be shortened, with any supplementary information relocated after the header?